apps/omg_watcher/lib/omg_watcher/db/txoutput.ex

```diff
@@ -211,6 +222,7 @@ defmodule OMG.Watcher.DB.TxOutput do
   end
 
   @spec get_sorted_grouped_utxos(OMG.Crypto.address_t()) :: %{OMG.Crypto.address_t() => list(%__MODULE__{})}
+  @decorate measure_event()
```

Comment:
wouldn't it make more sense, to measure this event in `OMG.Watcher.Web.Controller.Transaction.create/2`?

Reply:
it catches by phoenix (and make transaction with namespace web)

Reply:
but how does the value differ if this is actually the first thing it calls? why do we need to measure it separately?

Reply:
thanks to the appsignal event, we can divide the time to execute appsignal transaction into smaller parts and check what really takes us the most time